Add audit config option for metricsServer (#169)
kedacore/keda-olm-operator: README.md

    # Array of strings (format is either with prefix '--key=value' or just 'value')
    # args: []

    ## Audit Config

>     # Define basic arguments for auditing log files. If needed, more complex flags

Reviewer: Could you please add a link here to Kubernetes documentation about this?

    # Array of strings (format is either with prefix '--key=value' or just 'value')
    # args: []

    ## Audit Config
    # https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy

>     # Define basic arguments for auditing log files. If needed, more complex flags

Reviewer: would be nice to specify here mandatory fields, so users now which they need to specify.

    # can be set via 'Args' field manually.
    # auditConfig:
    #   logFormat: "json"

>     #   LogOutputVolumeClaim: ""

Reviewer: ```suggestion
    #   logOutputVolumeClaim: ""
```

Reviewer: would be nice to document a behavior here, where the log is stored if this option si specifed and what should be actually specified here?

    #   policy:
    #     rules:
    #     - level: Metadata
    #     omitStages: "RequestReceived"
    #     omitManagedFields: false
    #   lifetime:
    #     maxAge: ""
    #     maxBackup: ""

>     #     maxSize: ""

Reviewer: would be nice to write some example values here instead of `""`

    ## Audit Config
    # https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy
    # Define basic arguments for auditing log files. If needed, more complex flags

>     # can be set via 'Args' field manually. Non-empty 'policy' field is required
>     # for logging. If 'logOutputVolumeClaim' is empty log to stdout otherwise it
>     # is a name of user defined PersistentVolumeClaim resourse.

Reviewer: ```suggestion
    # can be set via 'Args' field manually. 
    # Non-empty 'policy' field is mandatory to enable audit logging.
    # If 'logOutputVolumeClaim' is empty the audit log is printed to stdout, otherwise it
    # point to the user defined PersistentVolumeClaim resource name.
```

    # auditConfig:
    #   logFormat: "json"

>     #   logOutputVolumeClaim: "persistentVolumeClaim"

Reviewer: ```suggestion
    #   logOutputVolumeClaim: "persistentVolumeClaimName"
```
